docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -955,17 +955,20 @@ Thankfully, the HL7 FHIR specification defines a way to express a dependency bet
 }
 ```
 
-In order to reference a resource via a local reference, you must first define a local identifier for that resource by specifying the `fullUrl` field within the request entry. To define a local identifier, use a value that starts with `urn:` as in the preceding example. The HL7 FHIR specification highly recommends that the value of the identifier should be a UUID-type value, although the FHIR server does not require the use of a UUID value. You can use any value you like after the `urn:` prefix as long as it is unique within the bundle. For example, you can use urn:Patient_1, urn:ABC, or urn:Foo. Just make sure that no two request entries share the same fullUrl value within the request bundle.
+To define a local identifier for a resource, you must specify it in the `Bundle.entry.fullUrl` field of the resource's bundle entry. The HL7 FHIR specification recommends that the local identifier be the persistent identity of the resource if known, otherwise a UUID-type value prefixed with `urn:` as in the preceding example. However, the FHIR server does not require the use of a UUID value. You can use any value you like after the `urn:` prefix as long as it is unique within the bundle. For example, you can use `urn:Patient_1`, `urn:ABC`, or `urn:Foo`. Just make sure that no two request entries share the same `fullUrl` value within the request bundle.
```

Comment:
```suggestion
To define a local identifier for a resource, you must specify it in the `Bundle.entry.fullUrl` field of the resource's bundle entry. The HL7 FHIR specification recommends that the local identifier be the persistent identity of the resource if known, otherwise a UUID-type value prefixed with `urn:uuid:` as in the preceding example. However, the FHIR server does not require the use of a UUID value. You can use any fullUrl value you like as long as it is unique within the bundle. For example, you can use `urn:Patient_1`, `urn:ABC`, or `urn:Foo`.
```

Reply:
Done.